content/300-guides/200-deployment/001-deploying-prisma.mdx

```diff
@@ -1,8 +1,8 @@
 ---
-title: 'Deployment overview'
+title: 'Deploying Prisma'
 metaTitle: 'Deploying projects using Prisma to the cloud'
```

Comment:
"to the cloud" is a bit weird maybe?
<img width="768" alt="image" src="https://github.com/prisma/docs/assets/183673/ab5dffe5-6286-4c94-86a4-e4dc848925ca">
Stumbled over it on this Google search result